src/dlgprefreplaygaindlg.ui

```diff
@@ -258,7 +264,7 @@
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
-         <string>The normal average ReplayGain playback level is -14 dB. You may increase it using the inital boost, if your tracks does not require the full dynamic headroom, or reduce it if you experience that your tracks are clipping by default.</string>
+         <string>The loudness of the ReplayGain reference signal is -18 LUFS (Loudness Units relative to Full Scale). You may increase it, if your tracks does not require the full dynamic headroom or reduce it if you experience that your tracks are clipping by default. The standard loudness for European TV is -23 LUFS and for podcasting it is -16 LUFS.</string>
```

Comment:
I do not have a handy link, that suits here. Since TV is an unlikely use-case for Mixxx, I will give only the -16 LUFS podcasting reference.